_episodes/04-loops.md

```diff
@@ -97,7 +97,7 @@ the list only included these four items, the shell exits the `for` loop at that
 >
 > Here we see `>` being used as a shell prompt, but `>` can also be
 > used to redirect output from a command (i.e. send it somewhere else, such as to a file, instead of displaying the output in the terminal) ---
-> we'll use redirection in [episode 5](/lc-shell/05-counting-mining).
+> we'll use redirection in [episode 5]({{ page.root }}/05-counting-mining/).

```

Comment:
the preferred way of doing links in episode files is `({{ page.root }}{% link <name of file> %})`.
The `{% link %}` command ensures that the file exists and the the generated link is functioning. It's best to not use "/episode-name/" format as such links don't work when using the lesson offiline.